recipes/gauxc/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set name = "gauxc" %}
+{% set version = "1.0" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/wavefunction91/gauxc/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: dee246c8329cf48be4b2938776016196076e4c998ca39ee5f2e35eaac9569e68
+    patches:
+      - patches/gh-152.patch  # Allow building with Clang 19
+      - patches/gh-156.patch  # Allow building with Apple Clang
+      - patches/gh-160.patch  # Allow building shared libs
+      - patches/gh-161.patch  # Allow finding highfive library
+  - url: https://github.com/wavefunction91/linalg-cmake-modules/archive/9d2c273a671d6811e9fd432f6a4fa3d915b144b8.tar.gz
+    sha256: 6e49c6c7c7df14b956d1fee77d9b6aad9ded3210cd639133c69a52b2e5ce4eae
+    folder: external/linalg-cmake-modules
+  - url: https://github.com/wavefunction91/IntegratorXX/archive/1369be58d7a3235dac36d75dd964fef058830622.tar.gz
+    sha256: a2b168ba8721cf4969f2324dd363a52698f3f2a3b71ef451e1d72cce367b002a
+    folder: external/integratorxx
+
+{% if mpi == "nompi" %}
+{% set build = build + 100 %}
+{% endif %}
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
+
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }} {{ mpi_prefix }}_*
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake >3.20,<4
+    - ninja
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - exchcxx >=1.0,<2
+    - gau2grid >=2.0.6,<3
+    - hdf5 * {{ mpi_prefix }}_*
+    - highfive >=2.4.1,<3
+    - libblas  # [linux]
+    - {{ mpi }}  # [mpi != "nompi"]
+  run:
+    - {{ mpi }}  # [mpi != "nompi"]
```

```suggestion
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - {{ stdlib('c') }}
    - cmake >3.20,<4
    - ninja
    - llvm-openmp  # [osx]
  host:
    - exchcxx >=1.0,<2
    - gau2grid >=2.0.6,<3
    - hdf5 * {{ mpi_prefix }}_*
    - highfive >=2.4.1,<3
    - libblas  # [linux]
    - libgomp  # [linux]
    - llvm-openmp  # [osx]
    - {{ mpi }}  # [mpi != "nompi"]
  run:
    - llvm-openmp  # [osx]
    - {{ mpi }}  # [mpi != "nompi"]
```

This is the correct way to add the dependency on OpenMP - on Linux, `libgomp` only needs to be in host, while on macOS, `llvm-openmp` needs to be in all three of build, host and run.  The reasons for this are idiosyncratic and will hopefully be simplified one day, but this is what is currently needed.